### ISSUE TITLE
Feature Request: Increase Volume Control Steps

### ISSUE THREAD

**Issue by danielniccoli** [punksta/volume_control_android#34]
It would be great to add the possibility to change the volume control steps.

What I mean by this is, fine tuning the percentage points that each button press of the volume control raises or lowers the volume.

In the following image you can see that the **media** slider has 15 steps, meaning that each volume control step translates roughly to a 6.67 % volume increase or decrease. My Google Pixel 3 has 25 steps (4 % increase/decrease). It would be amazing, if I would have the option to change this to e.g., 50 or 100 which translates to 2 % and 1% respectively.

This would allow for a much more fine tuned audio volume control, especially on devices where this is severely limited to 25 steps or less.

![image](https://user-images.githubusercontent.com/2971735/148684625-f762bffa-0a0a-4bde-90a7-63e816cd067e.png)

**Comment by DI555** [comment]
should be the option increase or DECREASE !

**Comment by usergithubbb** [comment]
Dear Stas,

It would be very nice if the plus and minus buttons of the Android notification widget could be lowered or raised one by one.  The jump by two is too big when listening to music.

**Comment by snan** [comment]
Yep, mine is way too loud at step three and way too quiet at step two and why there are fifteen steps total, thirteen of which are way too loud, is beyond me. So I really hope this is possible. 🤞🏻

Actually I'm gonna suggest a tweak. I realize that this is coming across as entitled so please don't misread it as a demand or expectation, just a blue sky wish. It's incredibly hard to set a slider to a very precise volume. I'd like either type-in with decimals, or + - buttons of both coarse and fine granularity.

I'm using thumb key and it has enough granularity but I wanted to set the slide length to 2 out of like 50 and it was super finicky and difficult to do that with Android's slider widget which is not good for very fine grained settings. If it's about quickly setting a good volume along a scale that's properly normalized from "just too quiet" on one end to "just too loud" on the other end and a choice of good options in between, then a slider widget is great, but it's not possible to normalize Android's volume settings in a way that works for all kinds of headphones.
